### DISCUSSION TITLE
api requests?

### DISCUSSION THREAD

**Discussion by fairlawn206** [supabase/supabase#14155] [Questions]
This is just a complete newbie question but supabase ai has no answers and I didn't see any here, so I apologize if it's ridiculous. I made a small little react app that allows to search a small club database of about 400 rows. single table in the supabase project. search by name, year etc functionality, ten columns in the table. very basic. it is only on my computer, testing it out from localhost. so all the api requests are coming from me.  This morning I did maybe 5 or 6 test searches. these generated 18,000 api requests. is this normal? did I set it up wrong? is it a problem? ignore? revise?

**Comment by GaryAustin1** [comment]
Sounds like you did something wrong.  But no way to know without code.
I recommend you go to the Supabase Discord channel as it is easy to work thru code issues/questions interactively there.

**Reply by fairlawn206** [reply]
thanks. how to get there? clicked on the link from supabase and nothing happened do I need an invite? sorry I don't know anything about discord either.

**Reply by GaryAustin1** [reply]
![image](https://user-images.githubusercontent.com/54564956/236517605-5306a95e-53dd-41fe-92ef-aedc163c6cdb.png)
You can also post some more info here, maybe it is obvious what is going.
There should be 1 API request per call to Supabase no matter how many rows are returned in the result.  So seems like you have a loop or a lot of re-rendering going on.

**Reply by fairlawn206** [reply]
I think I got it figured out. At least ChatGPT thinks so. Thanks for your help I appreciate it will check back if the thing keeps blowing up.

**Reply by GaryAustin1** [reply]
OK hopefully.  But chatGPT does not really know Supabase API syntax that well (it does fine at Postgres) because its training ended 2 years ago.
